WordPress/Sniff.php

```diff
@@ -42,37 +41,37 @@
 	 * @var array
 	 */
 	public static $escapingFunctions = array(
-		'absint' => true,
-		'esc_attr__' => true,
-		'esc_attr_e' => true,
-		'esc_attr_x' => true,
-		'esc_attr' => true,
-		'esc_html__' => true,
-		'esc_html_e' => true,
-		'esc_html_x' => true,
-		'esc_html' => true,
-		'esc_js' => true,
-		'esc_sql' => true,
-		'esc_textarea' => true,
-		'esc_url_raw' => true,
-		'esc_url' => true,
-		'filter_input' => true,
-		'filter_var' => true,
-		'intval' => true,
-		'json_encode' => true,
-		'like_escape' => true,
-		'number_format' => true,
-		'rawurlencode' => true,
-		'sanitize_html_class' => true,
-		'sanitize_user_field' => true,
-		'tag_escape' => true,
-		'urlencode_deep' => true,
-		'urlencode' => true,
-		'wp_json_encode' => true,
+		'absint'               => true,
+		'esc_attr__'           => true,
+		'esc_attr_e'           => true,
+		'esc_attr_x'           => true,
+		'esc_attr'             => true,
+		'esc_html__'           => true,
+		'esc_html_e'           => true,
+		'esc_html_x'           => true,
+		'esc_html'             => true,
+		'esc_js'               => true,
+		'esc_sql'              => true,
+		'esc_textarea'         => true,
+		'esc_url_raw'          => true,
+		'esc_url'              => true,
+		'filter_input'         => true,
+		'filter_var'           => true,
+		'intval'               => true,
+		'json_encode'          => true,
+		'like_escape'          => true,
+		'number_format'        => true,
+		'rawurlencode'         => true,
+		'sanitize_html_class'  => true,
+		'sanitize_user_field'  => true,
+		'tag_escape'           => true,
+		'urlencode_deep'       => true,
+		'urlencode'            => true,
+		'wp_json_encode'       => true,
 		'wp_kses_allowed_html' => true,
-		'wp_kses_data' => true,
-		'wp_kses_post' => true,
-		'wp_kses' => true,
+		'wp_kses_data'         => true,
+		'wp_kses_post'         => true,
+		'wp_kses'              => true,
```

Comment:
OK, we can keep them aligned. My issue isn't that it will be trouble to realign them (I use PHPStorm as well), but that doing so makes history harder to follow. But I guess it does make sense to prioritize the most common use, which is devs looking at the lists just as they are.